<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_prueba">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3dccab;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 25px; /* Bordes bastante redondeados */
    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); /* Sombra de fondo */
    transition: background-color 0.3s; /* Transición suave para el efecto hover */
}

QPushButton:hover {
    background-color: #35b197; /* Color oscurecido */
}</string>
    </property>
    <property name="text">
     <string>Inicar sesión</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_email">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingresa tu correo</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_error">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 12pt &quot;Sitka Banner&quot;;
	color: rgb(255,0,0)
	
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_pass">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Ingresa tu contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Inicio de sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>¿No tienes cuenta?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_registro">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>¡Registrate!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
